src/applications/verify/containers/VerifyApp.jsx

```diff
@@ -13,7 +13,10 @@ import { focusElement } from '~/platform/utilities/ui';
 export const selectCSP = selectedPolicy =>
   Object.values(SERVICE_PROVIDERS).find(csp => csp.policy === selectedPolicy);
 
-export const VerifyApp = ({ profile, useOAuth, loading }) => {
+export const VerifyApp = () => {
+  const loading = useSelector(isProfileLoading);
+  const profile = useSelector(selectProfile);
+  const useOAuth = useSelector(isAuthenticatedWithOAuth);
   useEffect(
     () => {
       if (!hasSession() || (hasSession() && profile.verified)) {
```

Comment:
@CaitHawk I'm going to talk to you more about this later on today when you have time. I'm thinking we can refactor this to be more concise, but just curious if should be in this PR or a new one 🤔